scripts/scripts.js

```diff
@@ -236,12 +268,14 @@ export async function loadBlock(block, eager = false) {
 /**
  * Loads JS and CSS for all blocks in a container element.
  * @param {Element} $main The container element
- * @returns {Array} of Promises to be resolved with blocks are loaded
  */
-export function loadBlocks($main) {
-  const blockPromises = [...$main.querySelectorAll('div.section-wrapper > div > .block')]
-    .map(($block) => loadBlock($block));
-  return blockPromises;
+export async function loadBlocks($main) {
+  const blocks = [...$main.querySelectorAll('div.block')];
+  for (let i = 0; i < blocks.length; i += 1) {
```

Comment:
Why not use `forEach`?

Reply:
Because of `await loadBlock(blocks[i])` in the loop. `await` in the `for(...)` loop will block the process, `forEach` will not block the process, thus you cannot know when `loadBlock`s will be executed.